HomeBloodGlucoseMonitor/R4/files/HomeBloodGlucoseMonitorOrderPrepopulation-0.0.1.cql

```diff
@@ -4,6 +4,8 @@ include FHIRHelpers version '4.0.0' called FHIRHelpers
 
 
 codesystem "HCPCS": 'https://bluebutton.cms.gov/resources/codesystem/hcpcs'
+codesystem "SNOMED-CT": 'http://snomed.info/sct'
```

Comment:
I don't see this is used in the file.

Reply:
Oh thanks for removing I created a test patient to test and make sure the functionality works as intended

Reply:
Do you have a PR for that patient in test-ehr?

Reply:
nope I don't I just got rid of the patient after it worked because there were already 2 patients that were there (found out after the fact)and tested the functionality as intended